pkg/skaffold/debug/transform_python_test.go

```diff
@@ -166,49 +167,66 @@ func TestPythonTransformer_Apply(t *testing.T) {
 			containerSpec: v1.Container{},
 			configuration: imageConfiguration{entrypoint: []string{"python"}},
 			result: v1.Container{
-				Command: []string{"python", "-mdebugpy", "--listen", "5678"},
+				Command: []string{"/dbg/python/launcher", "--mode", "debugpy", "--port", "5678", "--", "python"},
 				Ports:   []v1.ContainerPort{{Name: "dap", ContainerPort: 5678}},
-				Env:     []v1.EnvVar{{Name: "PYTHONUSERBASE", Value: "/dbg/python"}},
 			},
 			debugConfig: ContainerDebugConfiguration{Runtime: "python", Ports: map[string]uint32{"dap": 5678}},
 			image:       "python",
 		},
+		{
+			description:   "override protocol",
+			containerSpec: v1.Container{},
+			configuration: imageConfiguration{entrypoint: []string{"python"}},
+			result: v1.Container{
+				Command: []string{"/dbg/python/launcher", "--mode", "pydevd", "--port", "5678", "--", "python"},
+				Ports:   []v1.ContainerPort{{Name: "pydevd", ContainerPort: 5678}},
+			},
+			debugConfig:       ContainerDebugConfiguration{Runtime: "python", Ports: map[string]uint32{"pydevd": 5678}},
+			image:             "python",
+			overrideProtocols: []string{"pydevd"},
```

Comment:
WDYT about adding `,"dap"` here to ensure priority is handled correctly?
```suggestion
			overrideProtocols: []string{"pydevd","dap"},
```

We can then have a similar test but with the protocols reversed `"dap","pydevd"`.

Reply:
had to fix the actual python transform code to get this to pass, so thanks, good suggestion